pootle/apps/pootle_misc/checks.py

```diff
@@ -1106,6 +1104,31 @@ def get_qualitycheck_schema(path_obj=None):
     return result
 
 
+def get_qualitycheck_list(path_obj):
+    """
+    Returns list of checks sorted in alphabetical order
+    but having critical checks first.
+    """
+    result = []
+    checks = get_qualitychecks()
+
+    for check, cat in checks.items():
+        result.append({
+            'code': check,
+            'is_critical': cat == Category.CRITICAL,
+            'title': u"%s" % check_names.get(check, check),
+            'url': path_obj.get_translate_url(check=check)
+        })
+
+    def alphabetical_critical_first(item):
+        sort_prefix = 0 if item['is_critical'] else 1
+        return "%d%s" % (sort_prefix, item['title'])
```

Comment:
You can also use a tuple with `sorted()`:
```diff
-        sort_prefix = 0 if item['is_critical'] else 1
-        return "%d%s" % (sort_prefix, item['title'])
+        critical_first = 0 if item['is_critical'] else 1
+        return (critical_first, item['title'].lower())
```

Also note the necessary call to `lower()`, as it would currently display e.g. `Python` before `printf()`.